scripts/src/logseq/rewrite_clj.clj

```diff
@@ -0,0 +1,15 @@
+(ns logseq.rewrite-clj
+  "Rewrite-clj fns"
+  (:require [rewrite-clj.zip :as z]))
```

Comment:
This is a handy library and comes packaged with babashka. It really plays to Clojure's strength as code is data is code :)